site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+# Policy Attachment
+
+Many implementations of the API support additional functionality that is not
+covered by the API directly. For example, concepts like timeouts, retries, or
+even CDN configuration are not sufficiently portable to be defined within the
+Gateway API. Instead, implementations may choose to represent these concepts
+with custom Policy resources.
```

Comment:
It might be helpful for the reader to understand why these are not part of the API, cause that is the first question that comes to my mind.

Something like:
"While features like timeouts, x, y are present in most implementations, their details vary since there are no standards (RFCs) around them. This makes these features less portable. So instead of pulling these into the API, we offer a middle ground: a standard way to plug these features in the API and offer a uniform UX across implementations."

Reply:
Great idea, refactored the first paragraph to look like this.